templates/codehaus-mojo/External-Dependency-Info.txt

```diff
@@ -0,0 +1,222 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for GoogleGson incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# GoogleGson
```

Comment:
"GoogleGson"? Wrong license file?

Reply:
those are new bindings, so I forgot to change.

fixing